src/main/java/wordle/view/Console.java

```diff
@@ -0,0 +1,58 @@
+package wordle.view;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.StringJoiner;
+import wordle.model.Tiles;
+
+public class Console {
+
+    private final Scanner scanner = new Scanner(System.in);
+
+    public void printInitGameMessage() {
+        StringJoiner stringJoiner = new StringJoiner("\n");
+        stringJoiner.add("WORDLE을 6번 만에 맞춰 보세요.");
+        stringJoiner.add("시도의 결과는 타일의 색 변화로 나타납니다.");
+        System.out.println(stringJoiner);
+    }
+
+    public void printInputRequestMessage() {
+        System.out.println("정답을 입력해 주세요.");
+    }
+
+    public String inputAnswer() {
+        return scanner.nextLine();
+    }
+
+    public void printTiles(List<Tiles> tiles) {
+        for (Tiles tile : tiles) {
+            System.out.println(tile);
+        }
+    }
```

Comment:
개취인데 코틀린 러버로써 체이닝 메서드 사용하는걸 선호합니다!
지금 애플리케이션에선 큰 차이는 없지만 표준 출력을 한번만 사용하게 됨으로 표준 출력 블로킹에서 조금의 이점도 있을 것 같구요ㅎㅎ

딱구님은 어떻게 생각하시나요?

```suggestion
    public void printTiles(List<Tiles> tiles) {
        final String tilesString = tiles.stream()
                .map(Tiles::combine)
                .collect(Collectors.joining("\n"));
        System.out.println(tilesString);
    }
```

Reply:
너무 좋은것 같습니다 ㅋㅋ 특히, 표준 출력 블로킹을 줄일 수 있다는 관점 너무 좋네요 👍
반영 완료하였습니다 :) !!! 많이 배워가네요!!!